tests/providers/snowflake/operators/test_s3_to_snowflake.py

```diff
@@ -20,7 +20,7 @@
 from unittest import mock
```

Comment:
Can you also update [this tests](https://github.com/apache/airflow/blob/master/tests/test_project_structure.py#L55-L57)?

Reply:
Yop. Good catch.